packages/atomic/src/components/commerce/atomic-commerce-search-box/atomic-commerce-search-box.tsx

```diff
@@ -0,0 +1,691 @@
+import {isNullOrUndefined} from '@coveo/bueno';
+import {
+  SearchBoxOptions,
+  StandaloneSearchBox,
+  StandaloneSearchBoxState,
+} from '@coveo/headless';
+import {
+  SearchBox,
+  SearchBoxState,
+  buildSearchBox,
+} from '@coveo/headless/commerce';
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+  Prop,
+  State,
+  Watch,
+  h,
+} from '@stencil/core';
+import {AriaLiveRegion} from '../../../utils/accessibility-utils';
+import {hasKeyboard, isMacOS} from '../../../utils/device-utils';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {
+  SafeStorage,
+  StandaloneSearchBoxData,
+  StorageItems,
+} from '../../../utils/local-storage-utils';
+import {updateBreakpoints} from '../../../utils/replace-breakpoint';
+import {once, randomID} from '../../../utils/utils';
+import {SearchBoxWrapper} from '../../common/search-box/search-box-wrapper';
+import {SearchTextArea} from '../../common/search-box/search-text-area';
+import {TextAreaSubmitButton} from '../../common/search-box/text-area-submit-button';
+import {SuggestionManager} from '../../common/suggestions/suggestion-manager';
+import {
+  SearchBoxSuggestionElement,
+  SearchBoxSuggestionsBindings,
+  SearchBoxSuggestionsEvent,
+  elementHasQuery,
+} from '../../common/suggestions/suggestions-common';
+import {RedirectionPayload} from '../../search/atomic-search-box/redirection-payload';
+import {
+  ButtonSearchSuggestion,
+  SimpleSearchSuggestion,
+} from '../../search/atomic-search-box/search-suggestion';
+import type {Bindings} from '../atomic-commerce-search-interface/atomic-commerce-search-interface';
```

Comment:
I kept the same name as in the design doc, so you'll need to update this to atomic-commerce-interface. Also an alias for the binding unless you want to change to CommerceBindings in the rest of the file

```suggestion
import type {CommerceBindings as Bindings} from '../atomic-commerce-interface/atomic-commerce-interface';
```